packages/documentation/package.json

```diff
@@ -1,47 +1,26 @@
 {
-  "name": "documentation",
-  "version": "0.0.0",
-  "private": true,
+  "name": "rafiki-starlight",
+  "type": "module",
+  "version": "0.0.1",
   "scripts": {
-    "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "docusaurus build",
-    "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy",
-    "clear": "docusaurus clear",
-    "serve": "docusaurus serve",
-    "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc",
-    "docs:graphql:generate": "npx docusaurus docs:generate:graphql:backend-api && npx docusaurus docs:generate:graphql:auth-api"
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro",
+    "graphqlBackend": "./node_modules/.bin/graphql-markdown ../backend/src/graphql/schema.graphql > src/schema-dumps/backend_raw_schema.md",
```

Comment:
let's keep the commands as such:
`docs:graphql:generate`
`docs:graphql:generate:backend` 
`docs:graphql:generate:auth`

Just for consistency with the other times we generate things